Cargo.lock

```diff
@@ -2019,7 +2019,7 @@ dependencies = [
  "integritee-node-runtime",
  "ita-stf",
  "itc-rpc-client",
- "itp-api-client-extensions",
+ "itp-node-api-extensions",
```

Comment:
After discussing with @haerdib, I decided to rename the crate and move the node api factory into this crate (so it can be re-used)